MessengerOnFileForLinux/ChatControl/src/ConversationControl.cpp

```diff
@@ -107,17 +106,32 @@ bool ConversationControl::isMessagesToReadExist()
     }
 }
 
+void ConversationControl::handleInterlocutorInactivity() const
```

Comment:
co robimy dokładnie ta funkcja, by widać tak jakbym wysyłała wiadomość. Czy nie powinna robić to przez MessageFramework?

Reply:
Nie wysyłamy wiadomości, tylko printujemy informacje na ekranie. Nie widzę potrzeby żeby taka informajca znajdowała sie w pliku z rozmową

Reply:
Imho powinna znajdować się z pliku. Widzę to tak, profty z umieszczenia jej w pliku:
- dostajesz plik i widzisz już, że ktoś został wywalony
- ktoś kto skopiuje historię może pokazać drugiemu użytkownikowi, że go rozłączyło
- przyszłościowo pod ficzer z postmortemem, czyli jak Cię rozłączy to możesz wejść ponwnie do rozmowy by ją pobrać dla historii, jeśli nie zrobiłeś tego wcześniej, inaczej nikt by nie wiedział, że tak się skończyła rozmowa